pkgs/development/androidndk-pkgs/androidndk-pkgs.nix

```diff
@@ -63,6 +63,7 @@ rec {
     passthru = {
       inherit targetPrefix;
       isClang = true; # clang based cc, but bintools ld
+      hardeningUnsupportedFlags = llvmPackages.clang.cc.hardeningUnsupportedFlagsByTargetPlatform;
```

Comment:
Why not use something like https://github.com/NixOS/nixpkgs/pull/326166/commits/6cef413dfc05221bbae13befbc67f4a1d6ab00b2

Reply:
Wouldn't that mean we'd have to potentially update it later to disable other flags, whereas with the current code we can just go ahead and get rid of all the unsupported ones for the given arch at once?

Reply:
Yeah